converter/internal/staticconvert/testdata/prom_scrape.diags

```diff
@@ -1,2 +1,3 @@
 (Error) unsupported global evaluation_interval config was provided
+(Warning) Static mode command line flags which modify the running agent are not supported for the static to flow converter.
```

Comment:
Nice! Should we add some call to action / how to resolve it?
e.g. "Please review your promtail command line flags and ensure they are set in Flow mode config file where necessary."